src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -156,9 +156,7 @@ function MdAutocomplete () {
       // Set our variable for the link function above which runs later
       hasNotFoundTemplate = noItemsTemplate ? true : false;
 
-      if (attr.hasOwnProperty('tabindex')) {
-        element.attr('tabindex', '-1');
-      }
+      element.attr('tabindex', '-1');
```

Comment:
It looks like someone originally went to do the right thing and forgot a `!`. It _should_ be:
```
    if (!attrs['tabindex']) {
      $element.attr('tabindex', '-1');
    }
```